aries-site/src/pages/design-tokens/how-to-read-design-tokens.mdx

```diff
@@ -0,0 +1,100 @@
+| variant   | A stylistic variant of a token.                                                                                                                                                                                    | secondary, primary, critical, warning             | --                                                                                                                                                           |
+| element   | When a component has distinct pieces of anatomy, this refers to the specific sub-element.                                                                                                                          | item (in a menu), input (in a form field)         | --                                                                                                                                                           |
+| scale     | A size/emphasis modifier for primitive and semantic tokens.                                                                                                                                                        | strong, weak, 100, 200                            | --                                                                                                                                                           |
+| state     | Interactive or programmatically unique state.                                                                                                                                                                      | enabled, hover, readOnly, selected, indeterminate | --                                                                                                                                                           |
+| property  | An item’s style. This is required for component tokens.                                                                                                                                                            | background, borderRadius, lineHeight, fontSize    | It usually matches a CSS property, but it can also store other conceptual definitions such as size, paddingX, etc. For multi-word properties, use camelCase. |
+
+## Types of design tokens
+
+Each type of token uses a subset of the overall convention. To learn about types of tokens, read the documentation of [types of design tokens](/design-tokens/overview#types-of-design-tokens).
+
+### Primitive
+
+<Example height="auto">
+  <TokenHighlight highlight={['prefix', 'level', 'type', 'variant', 'scale']} />
+</Example>
+
+| prefix | level | type      | variant | scale | Token                    |
```

Comment:
Tables like these are very helpful. We might consider adding a row or two to each to solidfy the concept.

Reply:
Done.